examples/simple/posix/UB-PTHREAD6-bad.c

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include<stdio.h>
+#include<string.h>
+#include<pthread.h>
+#include<stdlib.h>
+
+
+
+pthread_t thread_id;
+pthread_mutex_t lock;
+
+void * locker_thread(void * ptr) 
+{
+    //unlocking an unlocked thread
```

Comment:
I am all for such comments, but we should aim at consistency: either we add such comments to all -bad examples, or to none, and we should either start all of them with a capital letter or none, and we should have a period at the end everywhere or nowhere.  actually I'm not very sure about the first, it kind of makes sense to add such comments selectively, where things are tricky.